Configure

```diff
@@ -223,25 +223,13 @@ if (grep /^reconf(igure)?$/, @argvcopy) {
 	    @{$configdata::config{perlargv}} : ();
 	die "Incorrect data to reconfigure, please do a normal configuration\n"
 	    if (grep(/^reconf/,@argvcopy));
-	$ENV{CROSS_COMPILE} = $configdata::config{cross_compile_prefix}
-	    if defined($configdata::config{cross_compile_prefix});
-	$ENV{CC} = $configdata::config{cc}
-	    if defined($configdata::config{cc});
-	$ENV{CXX} = $configdata::config{cxx}
-	    if defined($configdata::config{cxx});
-	$ENV{BUILDFILE} = $configdata::config{build_file}
-	    if defined($configdata::config{build_file});
-	$ENV{$local_config_envname} = $configdata::config{local_config_dir}
-	    if defined($configdata::config{local_config_dir});
+	$config{perlenv} = { defined($configdata::config{perlenv}) ?
+	    %{$configdata::config{perlenv}} : () };
```

Comment:
Why does one need outermost {}?

Reply:
I could do `$config{perlenv} = $configdata::config{perlenv}`, it's true...  experience has given me reason to be paranoid and to want to do a shallow copy of the hash rather than just copying the reference.

Reply:
I'm not really fond of this, because it's too subtle. I mean ideally code should not evoke more questions than necessary, especially such esoteric. So a commentary?

Reply:
I added documentation in `INSTALL`, does that work for you?

Reply:
I meant specifically subtle use of outer {} *here*. Comment elsewhere doesn't really cut it, because it's as problematic to make the connection to specific language syntax as unravel why it's used :-)

Reply:
I decided to be less paranoid instead ;-)